app/models/developer_key.rb

```diff
@@ -101,6 +101,14 @@ def account_name
     account.try(:name)
   end
 
+  def access_token_count
+    self.access_tokens.count
```

Comment:
I've got this implemented. It looks like it I need to call `reset_counters` on each `DeveloperKey` in order to get the initial counts populated for the existing keys. Would this be an appropriate use for a `DataFixup`-type migration?